config/routes.rb

```diff
@@ -81,7 +81,14 @@
 
   # You can have the root of your site routed with "root"
   # just remember to delete public/index.html.
-  root :to => 'home#index'
+
+  # MLN application supports multiple domains.
+  root :to => "home#index"
```

Comment:
Hi Deepika, I think if this line is here then the app will never reach line 87. I think the app needs to check for the constraint _first_, and have this as the general rule afterwards. So I'd move this line, `root :to => "home#index"` to line 92 (i.e., two lines after `match '/contacts-links' => ...` with a blank line inbetween.

Please double check that the routes still work as expected. Actually, could we create some unit tests to check which routes are invoked depending on the HOST header? Thanks!